src/app/static/src/app/store/root/actions.ts

```diff
@@ -22,6 +22,7 @@ export const actions: ActionTree<RootState, RootState> & RootActions = {
                 dispatch("surveyAndProgram/deleteAll")
             ]);
             commit({type: "Reset"});
+            commit({type: "load/LoadFailed", payload: {detail: "State is invalid. Resetting all steps."}});
```

Comment:
"State" and "resetting" are more our words than user facing ones. How about just "This file could not be loaded. Taking you back to step 1."

Maybe also "Please contact support..."?

I reckon we should also ditch the "Failed to load state." message in the modal, we already have "Load Error" as a title (and again I don't think "state" is meaningful to the user.)

Reply:
Since the other ticket I'm working on will potentially rescue some valid steps, I might make it something even more generic, "There was a problem loading your data. Some data may have been invalid. Please contact support etc.."